packages/api/lambdas/sf-event-sqs-to-db-records.js

```diff
@@ -77,30 +83,39 @@ const hasNoAsyncOpOrExists = async (cumulusMessage, knex) => {
   }, knex, tableNames.asyncOperations);
 };
 
-const getMessageCollection = async (cumulusMessage, knex) => {
+const getMessageCollectionCumulusId = async (cumulusMessage, knex) => {
   try {
     const collectionNameAndVersion = getCollectionNameAndVersionFromMessage(cumulusMessage);
     if (!collectionNameAndVersion) {
       throw new Error('Could not find collection name/version in message');
     }
-    return await knex(tableNames.collections).where(
+    const collection = await knex(tableNames.collections).where(
       collectionNameAndVersion
     ).first();
+    if (!isRecordDefined(collection)) {
+      throw new Error(`Could not find collection with params ${JSON.stringify(collectionNameAndVersion)}`);
+    }
+    return collection.cumulusId;
   } catch (error) {
     log.error(error);
     return undefined;
   }
 };
 
-const getMessageProvider = async (cumulusMessage, knex) => {
+const getMessageProviderCumulusId = async (cumulusMessage, knex) => {
   try {
     const providerId = getMessageProviderId(cumulusMessage);
     if (!providerId) {
       throw new Error('Could not find provider ID in message');
     }
-    return await knex(tableNames.providers).where({
+    const searchParams = {
       name: getMessageProviderId(cumulusMessage),
-    }).first();
+    };
+    const provider = await knex(tableNames.providers).where(searchParams).first();
+    if (!isRecordDefined(provider)) {
+      throw new Error(`Could not find provider with params ${JSON.stringify(searchParams)}`);
+    }
+    return provider.cumulusId;
```

Comment:
Since all I really want from these helpers is the cumulusId of the objects, if they exist, this refactor seemed cleaner to me

Reply:
That makes sense to me